lang/ui.ca.json

```diff
@@ -0,0 +1,1161 @@
+  },
+  "simulator-unmute": {
+    "defaultMessage": "Deixar de silenciar",
+    "description": "Aria label for the unmute simulator button"
+  },
+  "software-versions": {
+    "defaultMessage": "Versions de programari",
+    "description": "Heading for a table of software versions in the about dialog"
+  },
+  "start-coding-action": {
+    "defaultMessage": "Comença a codificar",
+    "description": "Start coding button text"
+  },
+  "support": {
+    "defaultMessage": "Suport",
+    "description": "Support menu option text"
+  },
+  "terms-of-use": {
+    "defaultMessage": "Condicions d'ús",
+    "description": "Terms of use menu option text"
+  },
+  "third-party-module-explanation": {
+    "defaultMessage": "Aquest fitxer és un mòdul de tercers i no està pensat per ser editat.",
+    "description": "Explanation shown instead of the code editor for third-party modules."
+  },
+  "third-party-module-how-to": {
+    "defaultMessage": "Permet editar mòduls de tercers a <link>Settings</link>  per modificar aquest mòdul.",
+    "description": "Explanation of how to allow editing for third-party modules."
+  },
+  "timeout-error-description": {
+    "defaultMessage": "No es pot connectar a la micro:bit",
+    "description": "Description for error messages relating to WebUSB timeout"
+  },
+  "timeout-error-title": {
+    "defaultMessage": "Temps de connexió esgotat",
+    "description": "Title for error messages relating to WebUSB timeout"
+  },
+  "toolkit-error-loading": {
+    "defaultMessage": "S'ha produït un error en carregar el conjunt d'eines.",
+    "description": "Error shown if we failed to load toolkit content"
+  },
+  "toolkit-view-documentation": {
+    "defaultMessage": "Consulta la documentació de {name}",
+    "description": "Aria label for the toolkit topic right arrow"
+  },
+  "transfer-hex-message-one": {
+    "defaultMessage": "Arrossega el fitxer hexadecimal de la teva <strong>carpeta de descàrregues</strong> a la <strong>unitat MICROBIT</strong>.",
+    "description": "Text in the transfer hex dialog"
+  },
+  "transfer-hex-message-two": {
+    "defaultMessage": "Pots <strong>obrir</strong> el fitxer hexadecimal més tard per continuar editant.",
+    "description": "Text in the transfer hex dialog"
+  },
+  "transfer-hex-title": {
+    "defaultMessage": "Transfereix el fitxer hexadecimal desat a la micro:bit",
+    "description": "Title for the transfer hex dialog"
+  },
+  "try-again-action": {
+    "defaultMessage": "Torna-ho a provar",
+    "description": "Try again button text"
+  },
+  "undo": {
+    "defaultMessage": "Desfer",
+    "description": "Aria label for the undo button"
+  },
+  "unexpected-error-description": {
+    "defaultMessage": "Si us plau, torna-ho a provar o <link>raise a support request</link> .",
+    "description": "Text shown for unexpected error scenarios"
+  },
+  "unexpected-error-title": {
+    "defaultMessage": "Si us plau, torna-ho a provar o <link>raise a support request</link>.",
```

Comment:
"raise a support request" needs translating

Reply:
requested in Crowdin